### DISCUSSION TITLE
启动报错<Consider defining a bean of type 'com.baomidou.dynamic.datasource.creator.DataSourceCreator' in your configuration.>

### DISCUSSION THREAD

**Discussion by teng-jy** [zuihou/zuihou-admin-cloud#166] [02.提问]
版本：4.9 lamp-cloud-pro-datasource-column
Mysql：8
问题：启动报错
详情：
***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 1 of constructor in top.tangyh.lamp.tenant.service.impl.DynamicDataSourceServiceImpl required a bean of type 'com.baomidou.dynamic.datasource.creator.DataSourceCreator' that could not be found.

Action:

Consider defining a bean of type 'com.baomidou.dynamic.datasource.creator.DataSourceCreator' in your configuration.

**Comment by zuihou** [comment]
nacos 中重新导入配置

**Reply by teng-jy** [reply]
请问 是导入 lamp-cloud-pro-datasource-column的配置吗？

**Reply by zuihou** [reply]
看着文档，用哪个项目，就导入哪个项目的配置，不同项目，配置不同！数据库不同！

**Reply by teng-jy** [reply]
好的 谢谢
